admin/build-doc

```diff
@@ -57,7 +57,7 @@ cd build-doc
 [ -z "$vdir" ] && vdir="$TOPDIR/build-doc/virtualenv"
 
 if [ ! -e $vdir ]; then
-    virtualenv --system-site-packages $vdir -p python2
+    virtualenv --system-site-packages $vdir
```

Comment:
we don't test builds at all for Python3, I take it you have been running this against Python3 and it succeeded?

Reply:
yes.